tests/boards/mec15xxevb_assy6853/i2c_api/README.txt

```diff
@@ -0,0 +1,23 @@
+I2C Mainline Test
+#################
+
+This application test is used to test the i2c driver api on microchip
+mec15xxevb_assy6853 board.
+
+
+Board Hardware Setup
+####################
+
+As the i2c slave device nxp pca95xx on mec15xxevb_assy6853 is connected
```

Comment:
```suggestion
As the I2C slave device NXP pca95xx on mec15xxevb_assy6853 is connected
```

Reply:
I have updated the README.txt as you suggested